### PR TITLE
corrected validation channel name

### PR DESCRIPTION
**Pull request by dingran**
Correct `validation` channel name from `test`

### CODE REVIEW

File: introduction_to_applying_machine_learning/ntm_20newsgroups_topic_modeling/ntm_20newsgroups_topic_model.ipynb [hunk 1]
```diff
@@ -1018,16 +1092,18 @@
...
    "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "conda_mxnet_p36",
+   "display_name": "Python 3",
    "language": "python",
-   "name": "conda_mxnet_p36"
+   "name": "python3"
```
Comment (yangaws) [inline introduction_to_applying_machine_learning/ntm_20newsgroups_topic_modeling/ntm_20newsgroups_topic_model.ipynb]: I think in SageMaker notebook there's not a kernel called 'Python 3' or 'python3'. The name should be 'conda_python3'. You could confirm that in other example notebooks.

File: introduction_to_applying_machine_learning/ntm_20newsgroups_topic_modeling/ntm_20newsgroups_topic_model.ipynb [hunk 2]
```diff
@@ -1039,7 +1115,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.4"
+   "version": "3.6.5"
   },
   "notice": "Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.  Licensed under the Apache License, Version 2.0 (the \"License\"). You may not use this file except in compliance with the License. A copy of the License is located at http://aws.amazon.com/apache2.0/ or in the \"license\" file accompanying this file. This file is distributed on an \"AS IS\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License."
```
Comment (yangaws) [inline introduction_to_applying_machine_learning/ntm_20newsgroups_topic_modeling/ntm_20newsgroups_topic_model.ipynb]: Copyright is outdated. Could you also check if other 1p notebooks are still in 2017 as well?

File: introduction_to_applying_machine_learning/ntm_20newsgroups_topic_modeling/ntm_20newsgroups_topic_model.ipynb (hunk 1 above)
Comment (dingran) [reply introduction_to_applying_machine_learning/ntm_20newsgroups_topic_modeling/ntm_20newsgroups_topic_model.ipynb]: will correct

File: introduction_to_applying_machine_learning/ntm_20newsgroups_topic_modeling/ntm_20newsgroups_topic_model.ipynb (hunk 2 above)
Comment (dingran) [reply introduction_to_applying_machine_learning/ntm_20newsgroups_topic_modeling/ntm_20newsgroups_topic_model.ipynb]: will copy over 2018 copyright notice

**Pull request merged**